src/module/classes/cleric.mjs

```
@@ -1,23 +1,43 @@
- *   @param {string} roll - The resulting roll.
+ * Adds functionality to Infused Healer.
+ *   @param {object} actor - The actor using the item.
+ *   @param {object} activity - The activity performed.
+ *   @param {object} usageConfig - The usage configuration.
  */
-export function healOver(actor,combatants){    
-    console.log("Nothing ")
+export function infuseHeal(activity, usageConfig) {
+    let actor = activity.actor;
+    if (activity.type === "heal" && actor.items.find(i => i.name === "Infused Healer")) {
+        const spellLevel = parseInt(usageConfig.spell.slot.replace('spell', ''));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)